util-images/containerd/Makefile

```diff
@@ -0,0 +1,17 @@
+PROJECT ?= k8s-testimages
+IMG = gcr.io/$(PROJECT)/perf-tests-util/containerd
+TAG = v0.0.1
+
+all: push
+
+.PHONY: build
+build: cmd
```

Comment:
There is no cmd dir here, let's make it depend on "Dockerfile"

Reply:
👍 done for both